app/decorators/case/overturned_ico/sar_decorator.rb

```diff
@@ -1,7 +1,15 @@
 class Case::OverturnedICO::SARDecorator < Case::OverturnedICO::BaseDecorator
 
+  def pretty_type
+    'ICO overturned (SAR)'
+  end
+
+  def requester_name_and_type
+    pretty_type
+  end
```

Comment:
Something feels a little off about this change. So far as I can tell, this just affects the display of this case on the "New Cases" page, is that right? And do we really just want to replace the requester name and type with the case type?

I can't find a design for this, or any tests, which is why I'm left wondering.

Reply:
@aliuk2012 points out that these came from the ICO Appeal decorators. I do think that this should be tidied up in another PR:

- add the case type to the New Cases page for other case types (i.e. FOI and SARs don't have the case type visible on the New cases page)
- fix the name of this method, if we don't change it, as we now have the case type in here
- add tests